pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidatesTest.scala

```diff
@@ -203,23 +203,14 @@ class SierraMergeCandidatesTest
         miroMergeCandidate(miroID) ++ miroMergeCandidate("V0013889")
     }
 
-    // - - - - - - -  Material type - - - - - - -
-    it("creates a merge candidate if the material type is '3D objects'") {
-      val bibData = create3DObjectWith(
-        varFields = create962subfieldsForWellcomeImageUrl(miroID)
-      )
-
-      SierraMergeCandidates(createSierraBibNumber, bibData) shouldBe
-        miroMergeCandidate(miroID)
-    }
-
-    it("creates a merge candidate if the material type is 'Digital Images'") {
+    it("Overrides non-suffixed IDs with suffixed ones where those are present") {
       val bibData = createPictureWith(
-        varFields = create962subfieldsForWellcomeImageUrl(miroID)
+        varFields = create089subfieldsWith(List("V0036036")) ++
+          create962subfieldsForWellcomeImageUrl("V0036036EL")
       )
 
-      SierraMergeCandidates(createSierraBibNumber, bibData) shouldBe
-        miroMergeCandidate(miroID)
+      SierraMergeCandidates(createSierraBibNumber, bibData) should contain theSameElementsAs
+        miroMergeCandidate("V0036036EL")
```

Comment:
I’d add a couple of extra test cases here:

* What if 089 and 962 have the same values?

    ```
    089 -> V0001234
    962 -> V0001234
    ```

* What if they're two different IDs, one with a suffix and one without:

    ```
    089 -> V0001234
    962 -> V0005678EL
    ```

I think I know what the desired behaviour is, but it would be good to (1) be explicit about that and (2) check it actually behaves that way.